.git-blame-ignore-revs

```diff
@@ -0,0 +1,48 @@
+# This file contains a list of git hashes of revisions recommended to be ignored by git blame.
+# These revisions are considered "unimportant" in that they are unlikely to be what you are
+# interested in when blaming.
+#
+# To ignore all such commits, change your git configuration as follows:
+#
+# git config --local blame.ignoreRevsFile .git-blame-ignore-revs
```

Comment:
I couldn't find a way to source-control this aspect of git config so that people don't need to run this command. I'd love to be proven wrong.

Reply:
Maybe adding this to `.git/config`? https://git-scm.com/docs/git-blame#Documentation/git-blame.txt-blameignoreRevsFile

Reply:
Nevermind, forgot `.git` is not versioned, oops 😄

Reply:
`.git` generally contains internal state related to the checkout (refs, blobs, HEAD pointer, any/all local config). I don't think we can expect to source-control the config file there. For example my local config file has a bunch of references to branches I work on.

Reply:
internally I'm sure all it's doing is passing it to git blame :P. You can specify it at that time with `--ignore-revs-file`. But I'm sure most people just want this enabled indefinitely.

Reply:
We could add it to a preinstall command or similar, but it feels hacky.